cmd/edit/machinepool/cmd.go

```diff
@@ -238,16 +276,92 @@ func run(cmd *cobra.Command, argv []string) {
 	}
 }
 
-func getReplicas(cmd *cobra.Command) (int, error) {
-	// Number of replicas:
-	replicas := args.replicas
-	if interactive.Enabled() || !cmd.Flags().Changed("replicas") {
-		return interactive.GetInt(interactive.Input{
+func getReplicas(cmd *cobra.Command,
+	reporter *rprtr.Object,
+	machinePoolID string,
+	existingAutoscaling *cmv1.MachinePoolAutoscaling) (autoscaling bool,
+	replicas, minReplicas, maxReplicas int) {
+
```

Comment:
The linter doesn't like this newline here...

Reply:
fixed